src/js/shared/constants.js

```diff
@@ -266,13 +266,67 @@ export const SEARCH_RESULT_OPTIONS = [
   },
 ];
 
+export const HEADER = {
```

Comment:
Since `TABS` and `LINKS` are _only_ used by the `header` component, you can just define them on that file (no need to be on `shared` if they are used in only one place).

-----

I know that on `options.js` you are using `HEADER.DEFAULT_TAB` to give a default value to `current`, **but** I think that on `header.svelte` you could do `export let currentTab = DEFAULT_TAB` and then on `options.js` you just do:

```js
    let current = _.get(params, "current");
    if(current) {
      headerComponent.$set({
        current,
      });
    } else {
       current = headerComponent.currentTab;
    }
```

Also on `updateVisibility` you can remove the `current` parameter and instead you can do:

```js
function updateTabsVisibility() {
  const current = headerComponent.currentTab;
  ...
}
```